dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_sref_precip_spatial_plots.sh

```diff
@@ -3,7 +3,7 @@
 #PBS -S /bin/bash
 #PBS -q dev
 #PBS -A VERF-DEV
-#PBS -l walltime=00:30:00
+#PBS -l walltime=00:10:00
```

Comment:
@BinbinZhou-NOAA The walltime in [dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_sref_precip_spatial_plots.sh](https://github.com/NOAA-EMC/EVS/pull/475/files#diff-e4d0aba2f35464b4cf2538f52e08a55ef2439bde4c1fca5d882ab60475da5106) went from 30 minutes, to 15 minutes, to 10 minutes. You told us by email that this job takes ~10 minutes to run. Can you please change the wallclock time back to 15 minutes?

CC @ShelleyMelchior-NOAA @PerryShafran-NOAA

Reply:
Binbin changed this to 10 minutes to match NCO ecf script. Marking as resolved!